parsing/location.mli

```diff
@@ -96,6 +96,9 @@ val rewrite_absolute_path: string -> string
         variable (https://reproducible-builds.org/specs/build-path-prefix-map/)
         if it is set. *)
 
+val get_build_path_prefix_map: unit -> Build_path_prefix_map.map option
+    (** Returns the map used by [rewrite_absolute_path]. *)
```

Comment:
I think that this should not be in `Location`. Could you maybe move the logic in utils/misc.ml, and then use it from `Location`? (I would rather keep it out of ` build_path_prefix_map.ml`, so that that one remains pure encoding/decoding.)